README.rst

```diff
@@ -37,127 +37,117 @@ the Rescorla-Wagner learning rule to these corpora.
 
-in order to build a source package run::
 
-    python setup.py sdist
+Documentation and Examples
+--------------------------
+Documentation and examples can be found under
+http://pyndl.readthedocs.io/en/latest/ or in the ``doc/`` folder after cloning
+the repository.
 
 
-Usage
-=====
-This package is intended to be used as a python package in (small) python
-script or via the ipython REPL. For example code and explanations look into
-``doc/source/examples.rst``.
+Running the tests
+=================
+If you have installed ``pyndl`` by cloning the repository change directory to
+the repository and run (for fast tests):
 
+.. code:: bash
 
-Development
-===========
+    cd pyndl
+    py.test
+    py.test doc/souce/example.rst
 
 
-Documentation
--------------
-The documentation and the doc-strings within the source code should follow the
-numpy doc-string conventions (which are used by pandas as well).
+For full tests you can run:
+
+.. code:: bash
+
+    tox
+
+For manually checking coding guidelines run:
+
+.. code:: bash
+
+    pep8 pyndl tests
+    pylint --ignore-patterns='.*\.so' --rcfile=setup.cfg -j 2 pyndl tests
+
+For more details on which tests are run in the continuous testing environment
+look at the file ``tox.ini``.
+
 
-https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt
+Deployment
+==========
+In order to create a source dist package run:
 
-http://pandas.pydata.org/pandas-docs/stable/contributing.html#contributing-to-the-documentation
+.. code:: bash
 
+    python setup.py sdist
+
+
+Contributing
+============
+Please read
+`CONTRIBUTING.rst
+<https://github.com/quantling/pyndl/blob/master/CONTRIBUTING.rst>`_ for details
+on our code of conduct, and the process for submitting pull requests to us.
 
-Install
--------
-You can install this package with pip from a folder, git repository, or sdist
-bundle::
 
-    pip install ~/pyndl/|git+ssh://git@github.com/<user>/pyndl.git|pyndl-<version>.tar.gz
+Versioning
+==========
+At the moment we are still in alpha and therefore no API garatuees are given,
+but soon we will change to use `SemVer <http://semver.org/>`_ for versioning. For
+the versions available,
+see the `tags on this repository <https://github.com/quantling/pyndl/tags>`_.
 
 
-Testing
+Authors
 =======
-Check if the package does what it is supposed to do by running the test with
-``py.test``::
+See also the list of `contributors
+<https://github.com/quantling/pyndl/contributors>`_ who participated in this
+project.
 
-    py.test-3 --cov-report html --cov=pyndl
 
-The more general way of testing - including unit tests, documentation checks
-and style checking - can be done via tox. Tox does testing in a virtual
-environment and installs all dependencies. ::
+License
+=======
+This project is licensed under the MIT License - see the `LICENSE.txt
+<https://github.com/quantling/pyndl/blob/master/LICENSE.txt>`_ file for details
 
-    (pip install tox)
-    tox
 
+Acknowledgments
```

Comment:
It is planned to publish ndl2 (at some point in the future) to github. I only need time for it. Probably in August.